docs/build-instructions/linux.md

```diff
@@ -29,8 +29,10 @@ Ubuntu LTS 12.04 64-bit is the recommended platform.
 
 ### Arch
 
-* `sudo pacman -S gconf base-devel git nodejs libgnome-keyring python2`
-* `export PYTHON=/usr/bin/python2` before building Atom.
+* To automatically build Atom from source, install the atom-editor package from the [Arch User Repository (AUR)](https://wiki.archlinux.org/index.php/Arch_User_Repository)
```

Comment:
I agree, the AUR is known to have its own set of issues, while the
atom-editor package is fairly stable from my experience, the AUR, in
general, is unreliable.

On 28 July 2015 at 11:07, Machisté N. Quintana <notifications@github.com>
wrote:

> In docs/build-instructions/linux.md
> <https://github.com/atom/atom/pull/8101#discussion_r35605982>:
>
> > @@ -29,8 +29,10 @@ Ubuntu LTS 12.04 64-bit is the recommended platform.
> >
> >  ### Arch
> >
> > -* `sudo pacman -S gconf base-devel git nodejs libgnome-keyring python2`
> > -* `export PYTHON=/usr/bin/python2` before building Atom.
> > +* To automatically build Atom from source, install the atom-editor package from the [Arch User Repository (AUR)](https://wiki.archlinux.org/index.php/Arch_User_Repository)
>
> We don't currently link to unofficial means of distributing Atom – would
> you mind taking out this line?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/atom/atom/pull/8101/files#r35605982>.
>



-- 
Thanks for your time,
Brenton